system/languages/en.php

```diff
@@ -97,6 +97,7 @@
 		"changed-css" => " edited the website's CSS layout",
 		"changed-html" => " edited the website's HTML structure",
 		"changed-page" => " edited the following page: ",
+		"deleted-page" => " deleted the following page: ",
```

Comment:
These changes add strings that are not used until the next commit, so they would make more sense in the next commit?

Reply:
Moved to other commit